iceoryx_posh/source/mepoo/memory_info.cpp

```diff
@@ -23,5 +23,5 @@ MemoryInfo::MemoryInfo(uint32_t deviceId, uint32_t memoryType)
     , memoryType(memoryType)
 {
 }
-}
-}
\ No newline at end of file
+} // namespace mepoo
+} // namespace iox
```

Comment:
New line missing?

Reply:
We do not need this since C++11. It is a requirement from the old C world which was abandoned with C++11.

Reply:
This comes from the POSIX standard where several tools like cat or wc threat only files with a newline at the end correctly. Otherwise it can lead to strange behaviour when the last line of the file is missing while processing it. I would vote for following this rule.